docs/reference/modules/ROOT/pages/building.adoc

```diff
@@ -1,20 +1,88 @@
+`crux-build` is a module within the crux repo with two key objectives:
 
-These can be found on the latest https://github.com/juxt/crux/releases[release] of Crux.
-Currently, these consist of a number of common configuration *uberjars* and a custom artifact builder.
+* Providing pre-configured JARs and Docker containers to start a basic implementation of Crux with no prior knowledge required.
+* Providing a mechanism to spin up custom Crux artifacts - with two necessary inputs:
+** A `deps.edn` file containing the dependencies that the node requires.
+** A `crux.edn` file, containing the configuration options to start the node with.
 
-To create your own custom artifacts for Crux:
+== Preconfigured Artifacts
 
-* Download and extract the `crux-builder.tar.gz` from the latest release
-* You can build an uberjar using either Clojure's `deps.edn` or Maven (whichever you're more comfortable with)
-** For Clojure, you can add further Crux dependencies in the `deps.edn` file, set the node config in crux.edn, and run `build-uberjar.sh`
-** For Maven, it's the same, but dependencies go in `pom.xml`
-* Additionally, you can build a Docker image using the `build-docker.sh` script in the **docker** directory.
+=== `crux-standalone`
 
-== Docker
+Available since `20.01-1.7.0-alpha`, on both:
 
-If you wish to use Crux with Docker (no JVM/JDK/Clojure install required!) we have the following:
+* https://hub.docker.com/r/juxt/[JUXT's Docker Hub].
+* https://github.com/juxt/crux/releases[GitHub releases], see the `crux-standalone.jar` attached to each release.
 
-* https://hub.docker.com/r/juxt/crux-standalone[*Crux HTTP Node*]: An image of a standalone Crux node (using a in memory `kv-store` by default) & HTTP server, useful if you wish to a *freestanding Crux node* accessible over HTTP, only having to use Docker.
+This starts up a basic, in-memory Crux node, utilising three Crux modules:
+
+* `crux-core` - required by all nodes.
+* `crux-cli` - required by all `crux-build` artifacts, used to start up the node and output configuration options to the command line.
+* `crux-http-server` - each configuration starts an HTTP server (on port 3000 by default).
+
+To learn more about using the Crux REST API, see the xref:http.adoc[HTTP docs].
+
+== Building Artifacts
+
+Alongside the JARs deployed on the GitHub releases is `crux-builder.tar.gz` - the scripts within this archive can be used to build a custom Crux JAR or Docker container.
+
+=== Building a JAR (Clojure CLI tooling)
+
+The `clj-uberjar` folder contains a number of files:
+
+* a `deps.edn` file to configure the Maven dependencies required
+* a `crux.edn` file to configure the node itself
+* a `resources/logback.xml` to configure logging output
+* a `build-uberjar.sh` script to build the JAR.
+
+Within this example, we override the default implementations to use `rocksdb` as the index store, document store and transaction-log store of the node (for more information, see xref:rocksdb.adoc[RocksDB]).
```

Comment:
Fine for a blog, probably a bit wordy for a reference - reference can be a lot more instructional/imperative in tone rather than conversational:

```
To use RocksDB as the ...:
* Add `crux-rocksdb` as a ...
+
[source,clojure]
...
* In `crux.edn`, ...
+
[source,clojure]
...
```